### ISSUE TITLE
[BUG] Solar Panels can be blocked by non-existent blocks

### ISSUE THREAD

**Issue by Quarri6343** [GregTechCEu/GregTech#844]
**Describe the bug**
this is not how GregTech power system work
you can always use a LV steam turbine to power MV machines without a transformer

this bug also makes 1V Solar Panel a decoration since there is no 1V battery buffer

**Versions**
master branch [3b1f165](https://github.com/GregTechCEu/GregTech/commit/3b1f1650c564c1c4852fa904d7355a7b1daabd40)

**Setup**
dev environment

**Comment by ALongStringOfNumbers** [comment]
What machines and what solars did you test this on, as it is working perfectly fine for me? I was able to charge an EV machine with an LV solar just fine.

**Comment by Quarri6343** [comment]
thanks for pointing it out!
I found this is a totally different issue

there are only circled solar panels working in the following screenshot:
![キャプチャ1240](https://user-images.githubusercontent.com/39122497/160220565-e2ebed98-86ab-4d15-ad2d-fea33efaf4b5.PNG)
I figured out the ghost blocks are blocking them

how to reproduce:
1)build a cleanroom in the mid air
2)switch the dev branch to the different one and reload the world
3)(cleanroom will be removed from the game)
4)build solar panels beneath the place where the cleanroom existed
5)solar panels are not working
6)reload the world or replace solar panels doesn't fix the issue

this is probably won't fix but I also think this should be archived

**Comment by Syrcan** [comment]
This is such a specific issue. I think it's more of a bug with removed blocks not being properly replaced with air.
